src/addTask.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-cycle */
+import { tasks, saveChanges, saveLocalstorage } from './dragDrop.js';
+import { tasksList } from './index.js';
+
+export default function addTask(task) {
+  const newTask = document.createElement('div');
+  newTask.classList.add('mini-section');
+  newTask.classList.add('item');
+  newTask.setAttribute('draggable', 'true');
+  newTask.innerHTML = `
+  <span>
+  <input class='check' type='checkbox' id='task-description' name='task-description' value='${task}'>${task}
+  <i class="fas fa-ellipsis-v ellipsis" style="color: gray; float: right;"></i>
+  </span>
+  `;
```

Comment:
[OPTIONAL] I suggest you use a constructor function with DOM instead of plain HTML in the future, in the way you make it more dynamically on future projects.